src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
```

Comment:
Not fixed 
![image](https://github.com/mate-academy/react_person/assets/70009205/7a50e5b5-2f6d-4d3a-9e7a-4a969c090eeb)

DRY - don't repeat yourself (avoid repeatable code: `person.`)
```suggestion
export const Person = ({ person }) => {
  const { name, age, sex, partnerName, isMarried } = person
```